app/channels/collect_submissions_channel.rb

```diff
@@ -0,0 +1,9 @@
+class CollectSubmissionsChannel < ApplicationCable::Channel
+  def subscribed
+    stream_for current_user
```

Comment:
Ah, one thing about security I forgot to mention: it would be good to enable action policy here, https://github.com/palkan/action_policy/blob/master/docs/rails.md#channels-integration. 

(You'll need to find and use the existing policy that governs whether a user can collect submissions or not.)

After you've enabled this, don't forget to add tests!